.gitignore

```diff
@@ -8,3 +8,4 @@ pkg/*
 bin/*
 files/**/cache/
 vendor/cookbooks
+files/chef-marketplace-gem/Gemfile.lock
```

Comment:
Gah - I did that for my own testing and it snuck through - I will pull that out right now.